public_html/wp-content/mu-plugins/blocks/blocks.php

```diff
@@ -103,6 +109,8 @@ function register_assets() {
  * Fix a CSS bug in Gutenberg when PanelRows are used with RangeControls.
  *
  * @todo Remove this when https://github.com/WordPress/gutenberg/pull/4564 is fixed.
+ *       it doesn't sound like they're going to accept any PRs for this, so we can probably remove ^ comment -- https://github.com/WordPress/gutenberg/issues/4402
+ *       it might also be good to refactor this to a more permenant solution, with a link to the ^ issue for documentation
```

Comment:
That's the impression I got from the comments on the issue/PR, but it does seem odd to me, since lots of plugins will have to add the same styles to avoid the problem. If you think they might accept a new PR then that sounds good to me.